tests/test_case_v2.json

```diff
@@ -0,0 +1,107 @@
+{
+    "functions": [
```

Comment:
Let's move to the same tools definition as openai to keep it consistent withe the tool calls and openai compatible.
e.g
tools: {
    type: function
    function: SCHEMA
}

Reply:
We need to change the dataset too, but thats the next step

Reply:
Fixed